tests/interfaces/test_jax_qnode.py

```diff
@@ -1334,6 +1334,9 @@ def test_hermitian(self, dev_name, diff_method, mode, shots):
         if diff_method == "backprop":
             pytest.skip("Backpropagation is unsupported if shots > 0.")
 
+        if diff_method == "adjoint" and mode == "forward":
+            pytest.skip("Computing the gradient for Hermitian is not supported with adjoint.")

```

Comment:
Gradients are computed on the forward pass with forward mode; skip such tests for adjoint with `qml.Hermitian`.